requirements.txt

```diff
@@ -2,7 +2,6 @@ tqdm>=4.66.2
 openai>=1.23.6
 tiktoken>=0.6.0
 transformers>=4.40.1
-pyserini>=0.43.0
```

Comment:
please add pandas, it is not here now because it gets installed as part of pyserini dependency